workflow/hosts/awspw.yaml

```diff
@@ -16,10 +17,11 @@ RESERVATION: ''
 CLUSTERS: ''
 CHGRP_RSTPROD: 'YES'
 CHGRP_CMD: 'chgrp rstprod' # TODO: This is not yet supported.
-HPSSARCH: 'YES'
+HPSSARCH: 'NO'
 HPSS_PROJECT: emc-global #TODO: See `ATARDIR` below.
+BASE_CPLIC: '/bucket/global-workflow-shared-data/ICSDIR/prototype_ICs'
```

Comment:
`BASE_CPLIC` is defined here for AWS.

Reply:
Yes. removed the AWS notice block from hosts/gfs_tasks.py